http/swagger.yml

```diff
@@ -4564,6 +4566,28 @@ components:
           type: string
         dialect:
           $ref: "#/components/schemas/Dialect"
+    FileASTNode:
+      description: flux File AST node
```

Comment:
I'd use the comment from the Flux code here:
https://github.com/influxdata/flux/blob/master/ast/ast.go#L186

Would you add descriptions for package, imports, name, and body using the godoc comments?